javalib/src/main/scala/java/util/Collections.scala

```diff
@@ -0,0 +1,1131 @@
+package java.util
+
+import java.{lang => jl, io}
+
+import scala.annotation.tailrec
+import scala.language.implicitConversions
+import scala.collection.JavaConversions._
+
+object Collections {
+
+  import WrappedCollections._
+  import ImmutableCollections._
+  import CheckedCollections._
+
+  final lazy val EMPTY_SET: Set[_] = {
+    unmodifiableSet(
```

Comment:
It is immutable, but it throws different exceptions when trying to use the iterator remove method.